### DISCUSSION TITLE
Leistungsabgabe oberhalb des Limits?

### DISCUSSION THREAD

**Discussion by GerhardSundt** [tbnobody/OpenDTU#2144] [Q&A]
Warum speist mein HM-1600 Leistung oberhalb des Limits ein? Die 405 W korrespondieren mit den Daten der Shelly- Plug Messung. Ich habe das temporäre Limit vom HA über Mqtt auf 26% (= 399 W) limitiert, die auch auf der OpenDTU page korrekt angezeigt werden. Trotzdem werden 405 W in’s Netz eingespeist. Die angegebenen Werte schwanken je nach Situation. Hat jemand eine Idee wie ich dem HM-1600 beibringen kann, sich an das Limit zu halten?
![IMG_0145](https://github.com/user-attachments/assets/5296f7d9-58cc-4505-8886-9891ae12715a)

**Comment by mroenne2022** [comment]
Das ist - glaube ich - normal, mein HM600 und mein HM1200 bringen im optimalen Fall auch immer ein paar % mehr.
Bespielsweise der HM600 bei 100% und 612W , der 1200er schon mal 1260W. Wenn du solch ein hartes Limit brauchst ggf. einfach 10W weniger einstellen !?
![Unbenannt](https://github.com/user-attachments/assets/a02704cb-d9ab-41df-91fe-8b59fd156dac)

**Reply by GerhardSundt** [reply]
Danke für Deine Beobachtung! 
Bei mir geht’s leider oft nicht um 10W, es geht manchmal um viel deutlicher abweichende Werte. Manchmal ist die Abweichung über einen Zeitraum von über 30 Sekunden > 2% (= > 30W) - manchmal auch mehr! Leider sehe ich überhaupt keine vernünftigen Grenzen für die Übersteuerung. So kann ich keine vernünftige 0- Einspeisung regeln! Siehe Screenshot nach dem Upgrade auf die neueste OpenDTU Firmware von heute, mit einer Abweichung von knapp über 1%. 😏
![IMG_0146](https://github.com/user-attachments/assets/955889b1-43ae-4167-a26b-6e502e02f049)

**Reply by GerhardSundt** [reply]
Sorry! Das waren sogar fast 80W über dem Limit…

**Comment by mroenne2022** [comment]
Ja ich seh , dazu hab ich dann keine Idee wie man das löst. Ich muss bei mir nicht begrenzen,  daher stört es micht nicht. Vielleicht liest ein Spezi mit der eine Erklärung dafür hat.
